index.html

```diff
@@ -13,154 +13,14 @@
 		margin: 20px;
 	}
 </style>
-<script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
```

Comment:
All of this was simply moved to stack-release.html. index.html is now a list of meeting agenda links. You may want to update existing links to go directly to the stack review meeting page, but you can still get to that page from here.